src/test/cypress/support/pageobjects/exercises/quiz/DragAndDropQuiz.ts

```diff
@@ -1,10 +1,37 @@
 import { EXERCISE_BASE, POST } from '../../../constants';
+import { MODELING_EDITOR_CANVAS } from '../modeling/ModelingEditor';
 
 export class DragAndDropQuiz {
     dragItemIntoDragArea(itemIndex: number) {
         cy.get('#drag-item-' + itemIndex).drag('#drop-location');
     }
 
+    setTitle(title: string) {
+        cy.get('#quiz-title').type(title);
+    }
+
+    dragUsingCoordinates(x: number, y: number) {
+        // @ts-ignore https://github.com/4teamwork/cypress-drag-drop/issues/103
+        cy.get('#modeling-editor-sidebar').children().eq(2).drag(MODELING_EDITOR_CANVAS, { target: { x, y } });
+        cy.wait(200);
+        cy.get(MODELING_EDITOR_CANVAS).trigger('pointerup');
+    }
+
+    getXAxis($els: JQuery<HTMLElement>) {
```

Comment:
I think it would be great if there is a JSDoc here to explain what this method does.